.github/workflows/create-bug-report.yml

```diff
@@ -9,8 +9,8 @@ jobs:
       - name: Extract version from branch name if release branch
         id: extract_version
         run: |
-          if [[ "$GITHUB_REF" =~ ^refs/heads/Version-v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
-            version="${GITHUB_REF#refs/heads/Version-v}"
+          if [[ "$GITHUB_REF" =~ ^refs/heads/(Version-v|release/v)[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            version="${GITHUB_REF#refs/heads/}"
```

Comment:
This will return `release/X.Y.Z` or `Version-vX.Y.Z` as the version, which is incorrect. We need to strip the prefix here, getting just the version number.